frontend/modules/dialog/dialog.redux.ts

```diff
@@ -104,20 +113,37 @@ export const showScreenshotDialog = (state = INITIAL_STATE, action) => {
 	return showDialog(state, {config});
 };
 
-export const hideDialog = (state = INITIAL_STATE) => {
+const showNewUpdateDialog = (state = INITIAL_STATE, action) => {
+	const config = {
+		title: 'Update Available',
+		template: Dialogs.NewUpdateDialogDialog,
+		onConfirm: action.config.onConfirm
+	};
+
+	return showDialog(state, { config });
+};
+
+const hideDialog = (state = INITIAL_STATE) => {
 	return { ...state, isOpen: false, isPending: false };
 };
 
-export const setPendingState = (state = INITIAL_STATE, {isPending}) => {
+const setPendingState = (state = INITIAL_STATE, {isPending}) => {
 	return { ...state, isPending };
 };
 
+const setMuteNotifications = (state = INITIAL_STATE, { muteNotifications }) => {
```

Comment:
I think we got rid of the dialog that uses this? Is it still needed?

Reply:
removed